classes/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] [REQUIRED] Your line 3 result should like this ``raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"``, please change it!